ingeniamotion/communication.py

```diff
@@ -1253,13 +1282,23 @@ def __load_ensemble_fw_ecat(self, net: EthercatNetwork, fw_file: str, slave: int
             net: Ethercat network.
             fw_file: Path to the ensemble FW file.
             slave: Slave ID (any slave in the ensemble)
+            boot_in_app: true if the bootloader is included in the application, false otherwise.
```

Comment:
You can explain what happens when it is None

Reply:
Done!